src/objectExplorer/objectTypes/treeNodeInfo.ts

```diff
@@ -0,0 +1,266 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Is this file a dupe of src/objectExplorer/nodes/treeNodeInfo.ts?  It looks awfully similar.